tests/rptest/clients/rpk_remote.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+
+class RpkRemoteTool:
```

Comment:
Would it be more flexible to have a utility called `RpkCommandBuilder` which constructs a command line argument and tests could use to build a command to run over ssh directly (and RpkTool could use too) than to have a separate wrapper just for remote rpk commands?

Reply:
I thought about something similar, but I think having tests that aren't node-local is valuable too, and mimics what a real user would do.